cmd/cdi-importer/importer.go

```diff
@@ -219,3 +211,12 @@ func main() {
 	}
 	klog.V(1).Infoln(message)
 }
+
+func convertQuantityToInt64(size resource.Quantity) int64 {
+	int64Size, asInt := size.AsInt64()
+	if !asInt {
```

Comment:
I'm not sure what this case is doing. Is this for 64-bit overflow?

Reply:
If I understand the internals of `Quantity`, it can be created as or converted to a Decimal form. My understanding is that in order to safely get `int64` from `Qunatity`, I have to deal with it.